Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM openjdk:8-alpine as builder
+
+LABEL maintainer="https://jbake.org/community/team.html"
+
+ENV JBAKE_HOME=/opt/jbake
+
+RUN mkdir -p ${JBAKE_HOME}
+COPY . /usr/src/jbake
+
+RUN set -o errexit -o nounset \
+    && echo "Building JBake" \
+    && cd /usr/src/jbake \
+    && ./gradlew installDist \
+    && cp -r jbake-dist/build/install/jbake/* $JBAKE_HOME \
+    && rm -r ~/.gradle /usr/src/jbake
+
+FROM openjdk:8-jre-alpine
+
+ENV JBAKE_USER=jbake
+ENV JBAKE_HOME=/opt/jbake
+ENV PATH ${JBAKE_HOME}/bin:$PATH
+
+RUN adduser -D -u 1000 -g "" ${JBAKE_USER} ${JBAKE_USER}
+
+USER ${JBAKE_USER}
+
+COPY --from=builder /opt/jbake /opt/jbake
+
+WORKDIR /mnt/site
+
+VOLUME ["/mnt/site"]
+
+ENTRYPOINT ["jbake"]
+CMD ["-b","-s"]
```

Comment:
Should we put `-s` in the command? Most of the use case could be run this as a part of build pipeline. 
To just bake the site without serving `-b` argument will need to be provided to override default CMD - 

```
docker run --rm -u jbake -v "$PWD":/mnt/site jbake/jbake:latest -b
```

Other option is to default it to "-b" but then users who will use it to server locally, will require to override CMD with `-b -s`.

Eitherway, I think we should document this in README.

Reply:
Yes. That's a good idea.

Reply:
Yeah good point, I'll default to just `-b` and update the README.